cmd/main.go

```diff
@@ -479,7 +479,7 @@ func getCommonCache(ctx context.Context, cli client.Client, platform common.Plat
 
 	namespaceConfigs[operatorNs] = cache.Config{}
 
-	if platform == cluster.ManagedRhoai {
```

Comment:
This is part of the "hacking to get the monitoring up" and shouldn't be merged.

Reply:
Yeah I understand I just left them in there to show anyone who was having a look at it